src/tss2-tcti/tctildr-nodl.c

```diff
@@ -49,37 +49,53 @@
 #include "util/log.h"
 
 #define ARRAY_SIZE(X) (sizeof(X)/sizeof(X[0]))
-#define NAME_ARRAY_SIZE 2
+#define NAME_ARRAY_SIZE 3
 
 struct {
-    const char *name;
+    const char *names [NAME_ARRAY_SIZE];
     TSS2_TCTI_INIT_FUNC init;
     char *conf;
     char *description;
 } tctis [] = {
 #ifdef _WIN32
     {
-        .name = "libtss2-tcti-tbs.so",
+        .names = {
+            "libtss2-tcti-tbs.so.0",
+            "libtss2-tcti-tbs.so",
```

Comment:
Isn't this superflous ?
If tbs is a name and the `handle_from_name()` will expand this to the .so.0 and .so files anyways, why not provide only `tbs` as the only option ?

Reply:
This code module is only built when `NO_DL` is defined. `handle_from_hame` is only defined when `NO_DL` is undefined.